### PR TITLE
Pretauadio

### PR DESCRIPTION
**Pull request by ltang85**
change dataspaces to mpi and verbose=4 to verbose=3

### CODE REVIEW

**Comment by hjjvandam** [comment]
These committed changes have a few problems. The main change is changing the ADIOS transport method from DATASPACES to MPI in sp_init.F (see adios_select_method call). All other changes are problematic:
- in md_main.F I have added calls to timer_name which associate labels to timers in the MD code, all those calls have been removed
- in md_main.F I have added a function md_adjvel which allows one to change the velocity of a set atoms so we can direct the dynamics to produce a result on a short time scale, this call has been removed
- in sp_init.F I have changed a number of format statements to use i7 instead of i5, these changes are needed to handle big systems such as the million atom case. The format statements changes were removed. 
In short it is much easier to make the relevant 1 line change from DATASPACES to MPI than to fix all the issues with this PR. Hence I have just committed the 1 line change and I will close this pull request. Nevertheless, this 1 line change will hopefully solve a lot of problems. Thanks for providing this critical piece of information.
